<launch>
  <master auto="start"/>
  
  <include file="$(find komodo_2dnav)/launch/remaper.launch" >
    <arg name="to" value="/diff_driver/command" />
    <arg name="from" value="/cmd_vel" />
  </include> 

  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <rosparam file="$(find komodo_coffee_taker)/config/gazebo/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find komodo_coffee_taker)/config/gazebo/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find komodo_coffee_taker)/config/gazebo/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find komodo_coffee_taker)/config/gazebo/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find komodo_coffee_taker)/config/gazebo/base_local_planner_params.yaml" command="load" />
  </node>
</launch>
